### PR TITLE
x0vncserver: add page

### PR DESCRIPTION
**Pull request by funwaa**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/screen.md [hunk 1]
```diff
@@ -33,3 +33,11 @@
...
 
 `screen -X -S {{session_name}} quit`
+
+- Attach screen that might still be attached elsewhere (detach remote):
+
+`screen -dr <name>`
+
+- Attach screen that is attached elsewhere (don't detach remote, will use resolution of remote's window):
+
+`screen -x <name>`
```
Comment (navarroaxel) [inline pages/common/screen.md:43]: The `md` pages can't have more than 8 examples, we have 10 here.

File: pages/linux/x0vncserver.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# x0vncserver
+
+> TigerVNC Server for X displays.
+> More information: <https://linux.die.net/man/1/x0vncserver>.
```
Comment (navarroaxel) [inline pages/linux/x0vncserver.md:4]: Here is the official docs
```suggestion
> More information: <https://tigervnc.org/doc/x0vncserver.html>.
```

File: pages/linux/x0vncserver.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+- Specify a passwordfile:
+
+`x0vncserver -display :0 -passwordfile <file>`
```
Comment (navarroaxel) [inline pages/linux/x0vncserver.md:8]:
```suggestion
`x0vncserver -display {{:0}} -passwordfile {{path/to/file}}`
```

File: pages/linux/x0vncserver.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+- Change the default port:
+
+`x0vncserver -display :0 -rfbport <port>`
```
Comment (navarroaxel) [inline pages/linux/x0vncserver.md:12]: We use `{{` for arguments:
```suggestion
`x0vncserver -display {{:0}} -rfbport {{port}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @funwaa, thanks for your submission! I left some comments.

If you think your examples are more important than the current ones in the `screen.md` please discuss about it. But we can't publish pages with more than 8 examples.

**Comment by funwaa** [comment]
I personally think that the -dr option for screen should be more in a cheatsheet than just "screen" without any options at all, but that's just my opinion. Thanks for the heads up and I'm sorry, I wasn't aware of the 8 commands maximum!

File: pages/linux/x0vncserver.md (hunk 3 above)
Comment (navarroaxel) [reply pages/linux/x0vncserver.md:8]: Hi @funwaa, you didn't apply 2 reviews.

**Comment by navarroaxel** [comment]
@funwaa Can you apply the changes to the `screen.md` page to continue with this PR?

File: pages/common/screen.md (hunk 1 above)
Comment (navarroaxel) [inline pages/common/screen.md:43]:
```suggestion
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/linux/x0vncserver.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# x0vncserver
+
+> TigerVNC Server for X displays.
+> More information: <https://tigervnc.org/doc/x0vncserver.html>.
+
+- Specify a passwordfile:
```
Comment (navarroaxel) [inline pages/linux/x0vncserver.md:6]:
```suggestion
- Start a VNC server with a passwordfile:
```

File: pages/linux/x0vncserver.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+`x0vncserver -display {{:0}} -passwordfile {{path/to/file}}`
+
+- Change the default port:
```
Comment (navarroaxel) [inline pages/linux/x0vncserver.md:10]:
```suggestion
- Start a VNC server using a specific port:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/linux/x0vncserver.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+# x0vncserver
+
+> TigerVNC Server for X displays.
+> More information: <https://tigervnc.org/doc/x0vncserver.html>.
+
+- Start a VNC server with a passwordfile:
```
Comment (navarroaxel) [inline pages/linux/x0vncserver.md:6]:
```suggestion
- Start a VNC server using a passwordfile:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @funwaa!

**Pull request merged**